tests/upgrade/0.8.2-0.9.0/jiva/volume_test.go

```diff
@@ -72,6 +78,39 @@ var _ = Describe("[jiva] TEST VOLUME UPGRADE", func() {
 			podCount := ops.GetPodRunningCountEventually("default", upgradedLabel, replicaCount+1)
 			Expect(podCount).To(Equal(replicaCount+1), "while checking pod version")
 
+			By("verifying registered replica count and replication factor")
+			podList, err := ops.PodClient.List(metav1.ListOptions{LabelSelector: ctrlLabel})
+			Expect(err).ShouldNot(HaveOccurred(), "while listing controller pod")
+
+			replicationFactor := ""
+			for _, env := range podList.Items[0].Spec.Containers[0].Env {
+				if env.Name == "REPLICATION_FACTOR" {
+					replicationFactor = env.Value
+				}
+			}
+			Expect(replicationFactor).ToNot(Equal(""), "while fetching replication factor")
+
+			curl := "curl http://localhost:9501/v1/volumes"
```

Comment:
Let us make use of go's standard http lib instead of curl

Reply:
@AmitKumarDas not sure how to use go's standard http lib since the test suite can't reach to the above ip, even if it is service ip (except for nodeport or load balancer type)
This can be possible if we run the test suite from inside a pod, then the service ip can be reachable.
Are we planning to package test suites into a pod?

Reply:
ok... we will use curl for now